appcatalog.go

```diff
@@ -0,0 +1,73 @@
+package appcatalog
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ghodss/yaml"
+	"github.com/giantswarm/microerror"
+)
+
+// GetLatestChart returns the latest chart tarball file in the AppCatalog storage URL
```

Comment:
```suggestion
// GetLatestChart returns the latest chart tarball file for the specified storage URL and app
```

This is good just suggesting a minor change.

Reply:
Nice, Thanks for noticing.